view/frontend/requirejs-config.js

```diff
@@ -1,5 +1,7 @@
 var config = {
-    "paths": {
-        "InstagramWidget": "Bitbull_InstagramWidget/js/InstagramWidget"
+    map: {
+        '*': {
+            InstagramWidget: 'Bitbull_InstagramWidget/js/InstagramWidget'
```

Comment:
Hi again, I couldn't find an official answer from M2 team on why map is the preference on M2, but from this  [link](http://stackoverflow.com/questions/19216580/requirejs-paths-vs-map/25199309) my understanding is map is the correct way to go.